CoroutineTemplate/app/build.gradle

```diff
@@ -67,8 +67,6 @@ android {
 }
 
 dependencies {
-    implementation project(':domain')
```

Comment:
ok, so you plan to separate `domain` module adding into the following PRs?

Reply:
Yeah, I think we can keep this module for a separate PR 

Because implementing it now for the sake of being able to import `AppError.kt` seems a bit much  🙏